### PR TITLE
STORY: Clean up the Mission Page #537

### LINKED ISSUE #537: STORY: Clean up the Mission Page
Mission Page Redesign

- [x] Clean up the layout
- [x] Improve readability
- [x] Improve object alignment
- [x] Rewrite the statements

### PR DESCRIPTION
**Pull request by stacysealky**
resolves #537

### CODE REVIEW

File: design/scss/custom/_mission.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+        h1 {
+            font-family: $font-heading;
+            border-bottom: 6px dotted $secondary;
+        }
+
+        h3 {
+            font-family: $font-body;
+            font-size: 1.15em;
+            max-width: 85%;
```
Comment (Enubia) [inline design/scss/custom/_mission.scss:13]: I'd recommend using some padding instead of a max-width
`.px-3.px-md-0`

File: design/scss/style.scss [hunk 2]
```diff
@@ -22,10 +22,6 @@ body {
 	align-items: center;
 }
 
-#header {
```
Comment (Enubia) [inline design/scss/style.scss:25]: why was this removed? it was used in some places

File: design/scss/_variables.scss [hunk 3]
```diff
@@ -24,14 +24,13 @@ $grays: (
...
 
 $primary: #333a64 !default;
 $secondary: #ff826b !default;
-$success: #00a19a !default;
-$warning: #ffbb59 !default;
-$danger: #ff5c6f !default;
-$info: #0dcaf0 !default;
+$success: #00a19a !default; // do not use unless discussed first
+$warning: #ffbb59 !default; // do not use unless discussed first
+$danger: #ff5c6f !default;
```
Comment (Enubia) [inline design/scss/_variables.scss:33]: we will probably use them for notifications, field validation, etc.

File: design/scss/_variables.scss [hunk 4]
```diff
@@ -24,14 +24,13 @@ $grays: (
...
 $navy-dark: #414254 !default;
-$cream: #f8e4c9 !default;
+$cream: #f8e4c9 !default; // do not use unless discussed first
 
 $primary: #333a64 !default;
 $secondary: #ff826b !default;
-$success: #00a19a !default;
-$warning: #ffbb59 !default;
-$danger: #ff5c6f !default;
-$info: #0dcaf0 !default;
```
Comment (Enubia) [inline design/scss/_variables.scss:34]: Is there a specific reason you removed this one?
It was a custom color I think and now applies the default bootstrap color which is darker than this one

File: views/pages/mission.pug [hunk 5]
```diff
@@ -4,54 +4,61 @@ block title
...
+                                span that collects your donations and sends holiday gifts and school supplies to foster children and youth at emergency shelters.
+                            p
+                                | Our verified partner agencies create and manage wish cards for the kids under their care.
+                                | Each wish card is unique and personal and linked with the gift item the child asked for.
+                                | Upon receiving your donation, we order and ship the gift item to each care agency.
+                                | The agency staffs or social workers collect and distribute the delivered items to each child.
+                .row.align-items-center
+                    .col-lg-6.col-12.px-5.d-flex.flex-column.justify-content-center.align-items-end
+                        .info-list
+                            h4 1. Transparent & Efficient
```
Comment (Enubia) [inline views/pages/mission.pug:33]: `&amp;` can be used here

File: views/pages/mission.pug [hunk 6]
```diff
@@ -4,54 +4,61 @@ block title
...
+                        img.img-fluid( src='/img/donate-school-supplies.jpg' alt='donate-school-supplies-to-children' loading='lazy' )
+
+                    .col-lg-6.col-12.p-5.d-flex.flex-column
+                        .info-list
+                            h4 3. Support Education
+                            p
+                                | Not only we donate holiday gifts, but we also host school supply donations.
+                                | Support the equality of educational opportunities by donating school supplies that the children need.
+
+                            h4.mt-5 4. Safe & Secure
```
Comment (Enubia) [inline views/pages/mission.pug:60]: `&amp;` can be used here

**Review by Enubia — CHANGES_REQUESTED** [review]

**Comment by stacysealky** [comment]
@Enubia - resolved PR comments and left replies for your questions
